easybuild/easyconfigs/v/vsc-mympirun-scoop/vsc-mympirun-scoop-3.5.0beta-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'VSCPythonPackage'
+
+name = 'vsc-mympirun-scoop'
+version = '3.5.0beta'
+
+homepage = 'http://hpcugent.github.com/vsc-mympirun/'
+description = """VSC-tools is a set of Python libraries and scripts that are commonly used within HPC-UGent."""
+
+# we build this to work with every python version
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+sources = ['scoop_0.7.tar.gz']
+source_urls = ['https://github.com/boegel/vsc-mympirun-scoop/archive/']
```

Comment:
WIP, https://github.com/hpcugent/vsc-mympirun-scoop/pull/8 should get merged first, and these have to fixed accordingly